README.md

```diff
@@ -1,60 +1,64 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![dbt logo and version](https://img.shields.io/static/v1?logo=dbt&label=dbt-version&message=0.21.x&color=orange)
+[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![dbt logo and version](https://img.shields.io/static/v1?logo=dbt&label=dbt-version&message=1.x&color=orange)
 
 # Readmissions 
 
-This is Tuva Health's Readmissions Mart, which is
-a dbt project to do hospital readmission analytics.
-It is based on the
-[CMS Readmission Measures Methodology](https://qualitynet.cms.gov/inpatient/measures/readmission/methodology)
-and includes other features,
-such as data quality checks specific to readmissions.
+Check out the [Tuva Data Model](https://docs.google.com/spreadsheets/d/1NuMEhcx6D6MSyZEQ6yk0LWU0HLvaeVma8S-5zhOnbcE/edit#gid=1991587675)
 
-## Links of interest
+Check out the latest [DAG](https://tuva-health.github.io/readmissions/#!/overview?g_v=1) for this concept
+
+Check out our [Docs](http://thetuvaproject.com/)
+
+Other links of interest related to this concept:
```

Comment:
This section was getting a little busy for readmissions. We may want to consider cleaning this up a bit. We could do something like this:

Check out:
 - the Tuva Data Model
 - our Knowledge Base
 - the latest DAG